tests/images.sh

```diff
@@ -50,7 +51,14 @@ call_for_fixtures() {
 if [ -n "${1:-}" ]; then
     # test with artifacts in cosa build dir
     basedir="$1"
-    if ! [ -e "${basedir}"/*.iso ] ;then
+    found=false
+    for f in "${basedir}"/*.iso; do
+        if [[ -e "${f}" ]]; then
```

Comment:
This could be reversed to "if doesn't exist" so `found` can be dropped.

Reply:
How would that work? If there is not file then we don't enter the for loop.

Reply:
Hum indeed!